extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -838,19 +839,29 @@ export class AmpStoryEmbeddedComponent {
       user().error(TAG, 'The tooltip icon url is invalid');
       return;
     }
-    const iconSrc = iconUrl ? parseUrlDeprecated(iconUrl).href :
-      embedConfig.componentIcon;
 
-    const existingTooltipIcon =
+    const tooltipCustomIcon =
       this.tooltip_.querySelector('.i-amphtml-story-tooltip-icon');
 
-    if (existingTooltipIcon.firstElementChild) {
-      addAttributesToElement(existingTooltipIcon.firstElementChild,
-          dict({'src': iconSrc}));
+    // Publisher specified a valid icon url.
+    if (iconUrl) {
+      this.resources_.mutateElement(devAssert(tooltipCustomIcon), () => {
+        setImportantStyles(devAssert(tooltipCustomIcon),
+            {'background-image': `url(${parseUrlDeprecated(iconUrl).href})`});
+      });
+      return;
+    }
+
+    // No icon src specified by publisher and no default icon in config.
+    if (embedConfig.componentIcon === BLANK_ICON_SRC) {
```

Comment:
Do we use `BLANK_ICON_SRC` for anything else? If not, should we just set `componentIcon` to `null`?